cli/cli/infra/cockpit.py

```diff
@@ -72,3 +93,83 @@ def create_metrics_token(api: sdk.CockpitV1Beta1API) -> str:
 def delete_metrics_token(api: sdk.CockpitV1Beta1API, token: sdk.Token) -> None:
     """Delete a Cockpit token."""
     api.delete_token(token_id=token.id)
+
+
+@contextmanager
+def temporary_grafana_user(
+    api: sdk.CockpitV1Beta1API,
+) -> Generator[sdk.GrafanaUser, None, None]:
+    """Create a temporary grafana user"""
+    user = None
+    try:
+        user = api.create_grafana_user(
+            login=GRAFANA_TEMPOARY_USER_LOGIN, role=sdk.GrafanaUserRole.EDITOR
+        )
+        yield user
+    finally:
+        if user:
+            api.delete_grafana_user(grafana_user_id=user.id)
+
+
+def get_metrics_data_source_uid(grafana_url: str, auth: HTTPBasicAuth) -> str:
+    """Get the data source uid for the Metrics data source
+
+    This data source is create by Cockpit for external data."""
+
+    resp = requests.get(
+        url=grafana_url + f"/api/datasources/name/{METRICS_DATASOURCE_NAME}",
+        auth=auth,
+        timeout=5,
+    )
+    resp.raise_for_status()
+
+    return resp.json().get("uid")
+
+
+def import_kong_statsd_dashboard(
+    api: sdk.CockpitV1Beta1API, user: sdk.GrafanaUser
+) -> str:
+    """Import the Kong StatsD dashboard into Grafana
+
+    Returns the url of the imported dashboard
+    """
+    url = get_grafana_url(api)
+
+    if not user.password:
+        raise RuntimeError(f"provided user {user.login} has no password")
+    basic = HTTPBasicAuth(username=user.login, password=user.password)
+
+    # We first get the dashboard from gnet as a json to import it
```

Comment:
In the Grafana UI you can just pass the ID to import a dashboard, is that not possible here?